validation/standard/standard-prestates.toml

```diff
@@ -1,3 +1,114 @@
-["op-contracts/v1.6.0"]
-program_version = "1.3.1"
-absolute_prestate = "0x038512e02c4c3f7bdaec27d00edf55b7155e0905301e1a88083e4e0a6764d54c"
+latest_stable = "1.4.0"
+latest_rc = "1.4.1-rc.2"
```

Comment:
Is it possible in the future we could have multiple latest_stable versions to call out here, one for each type? Or will there only ever be one type (cannon64 in this case) that can be considered latest_stable at a time. Wondering if we want a data structure here that would support latest stable for cannon64 + latest stable for kona + any other valid type?

Reply:
i think it would be multiple types per latest RC. we have the ability to modify this in the future if we need to.